templates/taxi/manufacturer_list.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     <h1>Manufacturer List
+      <a style="float: right" href="{% url 'taxi:manufacturer-create' %}">Create</a>
```

Comment:
Using inline styles for the 'Create' link is not recommended as it can lead to maintenance challenges. Consider using CSS classes instead to apply styles, which will make the code cleaner and more manageable.